cpp/velox/compute/VeloxColumnarToRowConverter.h

```diff
@@ -17,13 +17,17 @@
 
 #pragma once
 
+#include <arrow/c/abi.h>
```

Comment:
Do we need the abi.h in VeloxColumnarToRow?

Reply:
Fixed